src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/ExceptionTests.cs

```diff
@@ -553,10 +557,14 @@ void RunTest<T>(string json)
         [Fact]
         public static void SerializeUnsupportedType()
         {
-            RunTest<Type>(typeof(int));
-            RunTest<SerializationInfo>(new SerializationInfo(typeof(Type), new FormatterConverter()));
-            RunTest<IntPtr>((IntPtr)123);
-            RunTest<UIntPtr>((UIntPtr)123);
+            RunTest(typeof(int));
+            RunTest(new SerializationInfo(typeof(Type), new FormatterConverter()));
+            RunTest((IntPtr)123);
+            RunTest((UIntPtr)123);
+#if NETCOREAPP
+            RunTest(DateOnly.MaxValue);
+            RunTest(TimeOnly.MinValue);
```

Comment:
would it make sense to test for different value than min/max? I feel those two could pass test by accident

Reply:
Thought about it, but once implemented these would be perfectly valid serialization values. Same as DateTime.MaxValue and TimeSpan.MaxValue.